e2e/public-web/test_home_page.py

```
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)